about.html

```diff
@@ -12,11 +13,9 @@
     <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;1,300&display=swap"
         rel="stylesheet">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
-    <title>JavaScript Courses</title>
     <style>
         @import url('http://fonts.cdnfonts.com/css/cocogoose');
     </style>
-    <title>About</title>
 </head>
 <body>
     <header>
```

Comment:
- [ ] I kindly suggest that you add a top short bar section with e.g social media links, logout, and language links aligned to the right.

| Your Design  | Design Template |
| ------------- | ------------- |
| <img width="947" alt="navbar-1" src="https://user-images.githubusercontent.com/52934128/181600548-a78ad1c8-e4c0-42ec-bdc3-af1f1a03ae69.PNG">  | <img width="821" alt="navbar-2" src="https://user-images.githubusercontent.com/52934128/181600689-b1c93a7f-79b2-4984-b296-d1642666e91d.PNG"> |



- [ ] I kindly suggest that you add alt attribute description to your images on line number 24, 32, 36, 72, 83, 84, 91, 93, 99,  on your page as your alt attribute currently has empty string values in them.